lib/services/http-service.js

```diff
@@ -142,6 +144,17 @@ function httpServiceFactory(
             });
         }
 
+        //Swagger support
+        var swaggerConfig = {
+            appRoot: path.resolve(__dirname, '../../'),
+            configDir: path.resolve(__dirname, '../../swagger_config'),
+            swagger: path.resolve(__dirname, '../../static/monorail-2.0.yaml'),
```

Comment:
just to clarify,  the apidoc-swagger won't bee needed for 2.0 APIs since those will use swagger.  It's still needed to generate the swagger.json for existing 1.1 APIs (anything using apidoc annotation).